api/tests/src/api/competition_alpha/test_competition_route_get.py

```diff
@@ -35,64 +35,96 @@ def test_competition_get_200_with_api_key(client, api_auth_token, enable_factory
     assert response_competition["open_to_applicants"] == competition.open_to_applicants
 
 
-def test_competition_get_with_instructions_200(client, api_auth_token, enable_factory_create):
-    # Create a competition with instructions
+def test_competition_get_with_instructions_200(
+    client, api_auth_token, enable_factory_create, monkeypatch_session, mock_s3_bucket
```

Comment:
We don't want to use `monkeypatch_session` on an individual test, this changes the behavior for every single test that runs after it.

If we wanted to have the things you're setting be the defaults for all tests, that could be fine, but doing it this way is going to have weird side effects between tests.